<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>see_inventory</class>
 <widget class="QDialog" name="see_inventory">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_inventory">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>128</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>611</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
*{font-family:century gothic;
font-size:24px;
}
	
#pushButton_back
{
border-radius:10px;
	background-color:pink;
}
#pushButton_back:hover
{
background:#49ebff;
font-size:20px;
}
#pushButton_show_inventory
{
border-radius:30px;
	background-color:pink;
font-size:24px;
}
#pushButton_show_inventory:hover
{
background:#49ebff;
}
#lineEdit_password
{
 background-color:#333;
color:white
}
#lineEdit_card_number
{
background-color:#333;
color:white
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="pushButton_show_inventory">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>420</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>show inventory</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_back">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>550</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>back</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_password">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>290</y>
      <width>381</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_card_number">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>190</y>
      <width>401</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>card number</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
